test/test-suite/test_connection.py

```diff
@@ -102,8 +102,6 @@ def test_connection_matrix(self):
 
         assert (im - self.mono).abs().max() == 0
 
-    @skip_if_no("csvload_source")
-    @skip_if_no("csvsave_target")
```

Comment:
We can disable csv load and save with a meson option, so maybe this test should stay?

Reply:
There are no Meson options to disable CSV load/save, so we can assume it's always available. This is likely because the CSV loader must be explicitly called (since `->is_a_source` always returns `FALSE`), meaning it doesn't contribute to the potential attack surface.